config/crd/v1/bases/velero.io_backups.yaml

```diff
@@ -477,6 +477,15 @@ spec:
                 description: TTL is a time.Duration-parseable string describing how
                   long the Backup should be retained for.
                 type: string
+              uploaderConfig:
```

Comment:
Sorry for a late comment, didn't get a chance to review the design.
just wondering if we evaluated between passing this value as part of backup/restore API vs making it a server setting?
Any rationales around this. 
From my thinking, a user might not really wish to use different uploaderconfig for each backup and might just prefer to have it set once at server level. This might be sufficient for most use cases.

Reply:
@anshulahuja98 
Thank you for your suggestion; we have internally discussed this matter. Due to the various configuration options in Velero, the placement of these configurations is relatively unclear. We believe that some configurations can be placed at the server level, while others are associated with backup or restore. To clearly define the location of configurations, we have decided to place configurations related to backup or restore in the respective Backup or Restore Custom Resource (CR). Currently, aspects such as concurrency control and upcoming tasks like Kopia Sparse and Ignore Folder parameters are all linked to specific backup or restore operations and will therefore be placed in the corresponding CRs. Of course, we will also consider how to better distinguish where these configurations should be placed in the future.